### ISSUE TITLE
Color contrast not appropriate for error decorations in explorer

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#56785]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10
**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1)Navigate to Settings-->Select color theme -->Select Light (Visual studio)
2)Check color contrast ratio for the Explorer content on the screen.

**Actual:**
Color contrast ratio is less than 4.5:1 for the Red colored content when focused.
**Similar issue is observed in following themes:**
light Visual studio
Light+(Default light)
Quiet light
Solarized light
Abyss
Dark visual studio
Dark Default dark
Kimbie dark
Monokai
Monokai Dimmed
Red
Solarized dark
Tomorrow night blue

**Expected:**
Color contrast ratio should be more than 4.5:1 for the content on a screen.

Environment Details:
VSCode Version : 1.24.0
OS Version : Win10
Additional Details:
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer
Repro Steps:
1)Navigate to Settings-->Select color theme -->Select Light (Visual studio)
2)Check color contrast ratio for the Explorer content on the screen.
Actual:
Color contrast ratio is less than 4.5:1 for the Red colored content when focused.
Similar issue is observed in following themes:
light Visual studio
Light+(Default light)
Quiet light
Solarized light
Abyss
Dark visual studio
Dark Default dark
Kimbie dark
Monokai
Monokai Dimmed
Red
Solarized dark
Tomorrow night blue

**Expected:**
Color contrast ratio should be more than 4.5:1 for the content on a screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](url)

**User Impact:**
The low vision/visually impaired users will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - 
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
](url)

**Attachment For Reference:**
![22782_a11y_vscode_colortheme_colorcontrast](https://user-images.githubusercontent.com/39204909/44324660-f72e4880-a473-11e8-9a0a-3a7453dd7835.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
> **Environment Details:**
> VSCode Version : 1.24.0
> OS Version : Win10

This was addressed in version 1.26, this was tested using version 1.24. Please make sure you have the latest update when testing.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Checked and verified in below version i.e 1.27.0-Insiders.
The issue does not reproduced.
